Extraneous ' }) ' in file EventItem.tsx breaks docker build

### Describe the bug:
<!-- Please describe the bug in as much detail as possible. -->
At commit 22a34f1 in file EventItem.tsx there is an extra  `})` on line 66 that
is causing the build to fail.

```bash
node --version
# v14.16.0

npm --version
# 6.14.11

npx recursive-install

docker --version
# Docker version 19.03.5, build 633a0ea838

docker-compose --version
# docker-compose version 1.24.1, build 4667896

cat .env
# IS_DOCKER=TRUE

docker-compose build

# ./src/modules/dashboard/Events/components/EventItem.tsx:66:10
# Type error: Unexpected token. Did you mean `{'}'}` or `&rbrace;`?

#   64 |             {event.capacity}
#   65 |           </Typography>
# > 66 |         )}
#     |          ^
#  67 |         <Typography variant="body2" color="textSecondary" component="p">
#  68 |           {event.description}
#  69 |         </Typography>
# npm ERR! code ELIFECYCLE
# npm ERR! errno 1
# npm ERR! chapter-client@1.0.0 build: `next build`
# npm ERR! Exit status 1
# npm ERR! 
# npm ERR! Failed at the chapter-client@1.0.0 build script.

```
----

Just to see that this error could have been detected before building try:
```bash
npm run lint
# ...
# /home/northdecoder/workspace/chapter/client/src/modules/dashboard/Events/components
# /EventItem.tsx
#  66:9  error  Parsing error: Unexpected token. Did you mean `{'}'}` or `&rbrace;`?
...

```

### Tell us about your browser and operating system:
- Browser(s) name and version:  __terminal__
- Operating System: 
    ```bash
    uname -ior
    5.3.7-301.fc31.x86_64 x86_64 GNU/Linux
    ```

--- a/client/src/modules/dashboard/Events/components/EventItem.tsx
+++ b/client/src/modules/dashboard/Events/components/EventItem.tsx
@@ -63,23 +63,23 @@ const EventItem: React.FC<IEventItemProps> = ({ loading, event }) => {
           <Typography variant="body2" color="textSecondary" component="p">
             {event.capacity}
           </Typography>
-        )}
-        <Typography variant="body2" color="textSecondary" component="p">
-          {event.description}
-        </Typography>
-        {event.url && (
-          <Typography>
-            Event Url: <a href={event.url}>{event.url}</a>
+
+          <Typography variant="body2" color="textSecondary" component="p">
+            {event.description}
           </Typography>
-        )}
-        {event.video_url && (
-          <Typography>
-            Video Url: <a href={event.video_url}>{event.video_url}</a>
+          {event.url && (
+            <Typography>
+              Event Url: <a href={event.url}>{event.url}</a>
+            </Typography>
+          )}
+          {event.video_url && (
+            <Typography>
+              Video Url: <a href={event.video_url}>{event.video_url}</a>
+            </Typography>
+          )}
+          <Typography variant="body2" color="textSecondary" component="p">
+            {event.capacity}
           </Typography>
-        )}
-        <Typography variant="body2" color="textSecondary" component="p">
-          {event.capacity}
-        </Typography>
 
           <div>{event.tags && <Tags tags={event.tags} />}</div>
 

